src/components/RecipesList/RecipesList.jsx

```diff
@@ -1,16 +1,72 @@
+import { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+
 import RecipeItem from "../RecipeItem";
+import ModalWindow from "../ModalWindow";
+import RecipeModalContent from "../RecipeModalContent";
+
 import List from "./styles";
 
 const RecipesList = ({ recipes }) => {
+  const [showModal, setShowModal] = useState(false);
+  const [recipeID, setRecipeID] = useState(null);
+
+  const setModalData = (productData) => {
+    setRecipeID(productData);
+    setShowModal(true);
+  };
+
+  const mapApiResult = (result) => {
+    const mappedResult = result.map(({ recipe }) => ({
+      id: uuidv4(),
+      label: recipe.label,
```

Comment:
This data we can add just like
```
const mappedResult = result.map(({ recipe }) => ({
      id: uuidv4(),
      ...recipe
})

```

and prepare all info about recipe inside the RecipeItem